hack/build/resource-generator.sh

```diff
@@ -45,6 +45,7 @@ function generateResourceManifest() {
             -apiserver-image="${DOCKER_PREFIX}/${APISERVER_IMAGE_NAME}:${DOCKER_TAG}" \
             -uploadproxy-image="${DOCKER_PREFIX}/${UPLOADPROXY_IMAGE_NAME}:${DOCKER_TAG}" \
             -uploadserver-image="${DOCKER_PREFIX}/${UPLOADSERVER_IMAGE_NAME}:${DOCKER_TAG}" \
+            -ovirt-populator-image="${OVIRT_POPULATOR_IMAGE_NAME}" \
```

Comment:
`OVIRT_POPULATOR_IMAGE_NAME` should be "cdi-importer" at this point but that won't resolve to a proper image, no? what am I missing?

Reply:
yeah, for some reason the generated manifest still had the original image name, and was not updated. In my tests I changed the env var manually in openshift because HCO didn't properly update the manifests, not sure why though. Anyway, this part is a bit tricky for downstream

Reply:
> yeah, for some reason the generated manifest still had the original image name

The CI runs fine, which makes me question whether the forklift populator is in play at all?
I think one thing that is missing for sure is `DOCKER_PREFIX` here, so the image should be broken

Reply:
The controller doesn't run so nothing pulls this image I guess

Reply:
Doesn't the functionality in this PR get exercised in e2e?

Reply:
I am fine if not, can do in a follow up (if at all, it's just what we did with our internal flows, we converted existing DV operations to create populator CRs behind the scenes)

Reply:
> Doesn't the functionality in this PR get exercised in e2e? i.e. DataVolumes that used to do legacy now create the populator CRs?

I haven't added anything for it, so the controller probably runs, but the image won't be pulled because it doesn't invoke anything

I can try to add it later unless it's trivial, but doesn't look like it

Reply:
I've added some code to normalize the name of the image now

Reply:
Can't we just have the same treatment for it as the others?
`-uploadserver-image="${DOCKER_PREFIX}/${UPLOADSERVER_IMAGE_NAME}:${DOCKER_TAG}" \`

Reply:
it wouldn't be an issue with the downstream image? assuming it would receive a full external image URI

if not, then sure, we can do that

Reply:
I think if this works today for upload image etc should be fine for this one as well

Reply:
yes, but they are built as part of this project no?
my concern is that if we set OVIRT_IMAGE_POPULATOR=https://internal.registry/mtv/ovirt-populator:v4.16

then the URI would end up messed up, but I don't know what the downstream build process would look like so perhaps it's ok

Reply:
same here, maybe @tiraboschi (hi!) knows, anyhow - we can always fix the scripts to work with downstream

Reply:
ok, then I'll revert it back